src/test/java/com/google/sps/util/GmapsHelperTest.java

```diff
@@ -87,6 +89,20 @@ public void testSearchNearbyCategoryGroupReturnsValidJSON()
     Assert.assertEquals(true, isJSONValid(results));
   }
 
+  @Test
+  public void testrouteDistanceBetweenPoints()
```

Comment:
in your case, it could also be an empty list right? what will happen in that case? you probably want to add that test case too

Reply:
Yep, I'll add more test cases, thank you!

Reply:
Wait actually I just realized this is also an API call, so this shouldn't be part of Unit Tests right?